### PR TITLE
Update python supported versions

### PR DESCRIPTION
**Pull request by mshriver**
Major update to pytest, hold my beer

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +22,7 @@ jobs:
       deploy:
         provider: pypi
         skip_existing: true
+        # Ievgen
```
Comment (mshriver) [inline .travis.yml]: This scared me because I didn't recognize the username, so adding a comment.

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,41 @@
 [wheel]
-universal=1
+universal = 1
```
Comment (RonnyPfannschmidt) [inline setup.cfg]: This needs to go for py3 only

File: setup.cfg [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+
+[metadata]
+name = widgetastic.core
+author = Milan Falesnik
```
Comment (RonnyPfannschmidt) [inline setup.cfg]: We need to add maintainers, Milan no longer works on this

File: setup.cfg [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+author_email = mfalesni@redhat.com
+description = Making testing of UIs fantastic
+long_description= file: README.rst
+license = Apache license
+url = https://github.com/RedHatQE/widgetastic.core
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
```
Comment (digitronik) [inline setup.cfg]: @mshriver as we are supporting `py3.5`. can we include in `classifiers` and `tox` (It will skip if not found).

Comment (mshriver) [reply setup.cfg]: yes! sorry I removed it, then realized that pypy latest support was 3.5 and re-added. Thanks!

File: setup.cfg (hunk 3 above)
Comment (mshriver) [reply setup.cfg]: Agreed, will do.

File: .travis.yml (hunk 1 above)
Comment (izapolsk) [reply .travis.yml]: sorry, I didn't want to confuse you

File: setup.cfg (hunk 3 above)
Comment (mfalesni) [reply setup.cfg]: If you wanty, you can remove me from the `author` field and just add some "collective" in there, definitely for the e-mail (which is no longer valid), but I may start using this lib in my job again therefore possibly making contributions again.

Comment (quarckster) [reply setup.cfg]: I against that. The original author should stay here at least the name.

**Pull request merged**